web/client/product/appConfigDashboardEmbedded.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+*/
+
+import Dashboard from './pages/Dashboard';
+import version from '../reducers/version';
+import dashboardEpics from '../epics/dashboard';
+import widgetsEpics from '../epics/widgets';
+
+import dashboard from '../reducers/dashboard';
+import widgets from '../reducers/widgets';
+
+export default {
+    pages: [{
+        name: 'dashboard-embedded',
+        path: '/:did',
+        component: Dashboard,
+        pageConfig: {
+            name: 'dashboard-embedded'
+        }
+    }],
+    initialState: {
+        defaultState: {},
+        mobile: {}
+    },
+    baseReducers: {
+        version,
+        dashboard,
+        widgets
+    },
+    baseEpics: {
+        ...dashboardEpics,
```

Comment:
if these are included by the plugin, this are not needed here, right?

Reply:
I could remove them from the plugin and keep them only here.